README.md

```diff
@@ -116,12 +116,29 @@ The list of available actions can be seen via `medic-conf --help`.
 
 * upload of custom translations to the server
 
-## create-users **[ALPHA]**
+## create-users
 
-N.B. this feature is currently in development, and probably not ready for production yet.
 
 To create users on a remote server, use the `create-users` action.  The CSV file should be called `users.csv`, and an example is available [in the tests directory](test/data/create-users), for [an existing place](test/data/create-users/existing-place) and [a new place](test/data/create-users/new-place).
```

Comment:
The `csv-to-docs` documentation has excellent examples with sample csvs and the example outputs. I know this goes beyond your original intent of this change, but should we follow that same format instead of linking to these test folders?

Reply:
Yeah that makes sense. I'll write that up.